code/modules/mob/living/simple_animal/friendly/corgi.dm

```diff
@@ -167,3 +167,57 @@
 				for(var/i in list(1,2,4,8,4,2,1,2,4,8,4,2,1,2,4,8,4,2))
 					set_dir(i)
 					sleep(1)
+
+/mob/living/simple_animal/corgi/E_M
```

Comment:
E=Mc^2
?

Reply:
It is hound that chose the name

Reply:
It's E-N, it reads like Ian, it's a robotic corgi that exists on some other SS13 codebases.